### ISSUE TITLE
Activity links link to fromthepage.com addresses rather than relative paths

### ISSUE THREAD

**Issue by jackweinbender** [benwbrum/fromthepage#1460]
I noticed that the links to deeds on both the user profile ("recent activity") and dashboard (owner dash and collaborator dash under "your activity") link to the fromthepage.com host in my local environment. I'm guessing this has to do with a change I made a while back to use the `_url` helper methods (rather than `_path`) in the deed partial. We did this so that links in the mailers (which use these partials) would still work.

I don't *think* this is a serious problem, since local installs should configure their hostname in such a way that the right URLs will be generated. That said, I'm pretty sure the deed partial has some logic in it that is supposed to prevent this, so, we should try to figure it out.

**Comment by atomotic** [comment]
same behaviour for me
i've deployed fromthepage using this [docker](https://github.com/ryanfb/docker_fromthepage) behind nginx

these links are rendered with only_path="true"
is **app/views/deed/_deed.html.slim** the template to look?


```<a href="http://localhost:3000/andrea" only_path="true">andrea</a> transcribed
page
<a
  href="http://localhost:3000/raffaele/biblioteca-digitale-estense/trentasei-lettere-inedite-di-l-a-muratori/transcribe/6"
  only_path="true"
  >0006 - A.M.48.25</a
>
in
<a
  href="http://localhost:3000/raffaele/biblioteca-digitale-estense"
  only_path="true"
  >biblioteca digitale estense</a
>
collection
```

**Comment by bencomp** [comment]
I agree that installs should configure the hostname in the configuration. I guess @jackweinbender did not change the hostname?

I also see what @atomotic sees: `only_path="true"` in HTML elements. I don't understand most of Rails' magic, but I think <https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to> may provide hints, e.g. that the `only_path: !mailer` part of the `link_to` call should move inside the `..._url()` call.

A different issue that I see is that HTTP URLs are generated instead of HTTPS. I changed the `config.default_url_options` already to include `protocol: "https"` (as to me it looks like the automatic use of https when `config.force_ssl = true` only becomes available in Rails 5, whereas we currently use Rails 4.1.2), but it doesn't help.

**Comment by bencomp** [comment]
I'm learning a little about Rails routing magic :)

<https://stackoverflow.com/questions/11939865/named-routes-path-vs-url> explains the difference between `_url` and `_path` helpers: `_url` creates an absolute URL with schema, host and path, whereas `_path` creates a relative URL (unless you add `only_path: false`).

**Comment by bencomp** [comment]
Removing the initialiser resulted in having the defaults from Rails: `http_host: 'example.org', https: false`. So I think I will have to update the initialiser.

**Comment by benwbrum** [comment]
I think that this is resolved.
